apps/marginfi-v2-ui/src/components/mobile/MobileNavbar/MobileNavbar.tsx

```diff
@@ -27,7 +27,7 @@ export const mobileLinks: NavLinkInfo[] = [
     Icon: IconBuildingBank,
   },
   {
-    href: "/mint",
+    href: "/stake",
     alt: "mint icon",
     label: "mint",
```

Comment:
@masterzorgon this still says "mint" as the label. I believe you wanted to change everything to stake, correct?

Reply:
![image](https://github.com/mrgnlabs/mrgn-ts/assets/155211932/4fc52fcb-9c77-4320-96cd-f28b3466c829)

it should be resolved now